src/platforms/dart/migration.mdx

```diff
@@ -3,6 +3,16 @@ title: Migration Guide
 sidebar_order: 1000
 ---
 
+## Migrating from `sentry` `5.1.x` to `sentry` `6.0.0`
+
+* `Sentry.currentHub` is removed. Please use the static methods on `Sentry`
+* `SentryOptions.cacheDirSize` was renamed to `SentryOptions.maxCacheItems`
+* `EventProcessor` was changed from a callback to an interface
```

Comment:
missing entries:

Feat: Change timespans to Durations in SentryOptions (#504)
Feat: beforeSend callback accepts async code (#494)
Feat: Lists of exceptions and threads (#524)

adapt HTTP client examples because of Feat: HTTP breadcrumbs have the request & response size if available (#552) changes

we also have to fix every code example in dart.mdx and flutter.mdx that touches breaking changes, if any